### PR TITLE
Fixes percentage demo

### PR DESCRIPTION
**Pull request by nkgm**

### CODE REVIEW

File: tests/dummy/app/controllers/percentages.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default Ember.Controller.extend({
           this.set('columns', [20, 20, 40, 20]);
           break;
         case 3:
-          this.set('columns', [33, 33, 33]);
+          this.set('columns', [33, 33, 34]);
```
Comment (rwjblue) [inline tests/dummy/app/controllers/percentages.js]: Would `33.33, 33.33, 33.33` work instead?

**Comment by nkgm** [comment]
Any reason why CI is failing for this specific PR? Not getting any failures locally.

**Comment by rwjblue** [comment]
It may be related to my recent update to ember-cli@2.3.0-beta.2, can you rebase?

**Comment by nkgm** [comment]
Rebased. Times out after 13 minutes now. How long does it normally take to test against all framework versions?

**Comment by rwjblue** [comment]
Generally only a few minutes. Can you compare to other recent builds of master?

**Comment by nkgm** [comment]
Previous build got stuck 10 minutes on `releases/ember-2.3` scenario for no apparent reason. Probably a glitch.

**Comment by raytiley** [comment]
@nkgm thanks for fixing this. I'll make sure to get some acceptance tests in for the dummy app soon so I don't break it again. These commits should be squashed as I think they were only to trigger CI. Otherwise :+1:

**Comment by nkgm** [comment]
Thanks @raytiley. I'll squash them. I can do those acceptance tests if you don't mind providing some more feedback when that's done.

**Comment by raytiley** [comment]
Thanks @nkgm I squashed and merged this: https://github.com/emberjs/ember-collection/commit/d9fd7ef2da3a73e217c3767a03886d163cab528f

**Pull request closed without merging**